src/utils/stringify.ts

```diff
@@ -1,3 +1,7 @@
-export default function stringify(data: string) {
+export function stringify(data: string) {
 	return JSON.stringify(data.replace(/([^\\@#])?([@#])/g, '$1\\$2'));
```

Comment:
good catch! for a brief moment while I was bashing my head against a wall they had different regexes, hence the (now unnecessary) duplication. fixed